packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -171,10 +171,9 @@ export default class Dropdown extends React.Component {
       downshiftProps,
     } = this.props;
     const inline = type === 'inline';
-    const className = ({ isOpen }) =>
-      cx(`${prefix}--dropdown`, containerClassName, {
+    const className = () =>
```

Comment:
Seems that we no longer need to define `className` as a function (given there is no arg).

Reply:
good point thanks!

Reply:
@asudoh i'm actually getting an error when changing this statment to `const className = cx(…)`. it's getting close to EOD for me, so maybe i'm spacing out about something hehe…

Reply:
do you have any details on the error? did you change the function invocation on L231?

Reply:
@emyarod yeah you're right! thanks, i was blanking last night lol